### PR TITLE
Feature/products

### PR DESCRIPTION
**Pull request by JuanequeX**
Description
---
This section is for new products on the home page, which show 6 cards of products with title, category, price and a button to add cart.

### CODE REVIEW

File: app/javascript/components/products/Products.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      method: 'GET',
+      headers: {
+        'Authorization': 'Bearer 75be11a6e0d07b836aa2621c65f8936706499db21f0d0e5f',
+        'Content-Type': 'application/json'
+      }
+    })
+    .then(data => data.json())
+    .then(data => {
+      setProducts(data.products);
+      console.log(data);
```
Comment (OswaldoPineda) [inline app/javascript/components/products/Products.js]: pls remove this console log

File: app/javascript/components/products/Products.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <span className="products-header__subtitle">Ver mas</span>
+      </div>
+      <div className="cards-container">
+        { products.map((product) => <CardProduct product={product} />) }
+      </div>
+    </section>
+  );
+};
+
+  export default Products;
```
Comment (OswaldoPineda) [inline app/javascript/components/products/Products.js]: check the indentation here

File: app/javascript/components/products/cardProduct.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+.card-product {
+  border-radius: 8px;
+  border: 1px solid $black;
+  box-shadow: 3px 3px 6px 0 rgba(1, 1, 1, 0.29);
+  height: 189px;
+  margin-bottom: 20px;
+  width: 153px;
+
+  @media screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: check if we already have a variable for this resolution

File: app/javascript/components/products/cardProduct.scss [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+       display: block;
+    }
+  }
+
+  &__image {
+    background: $sand;
+    border-radius: 8px 8px 0 0;
+    height: 97px;
+
+    @media screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: check if we already have a variable for this resolution

File: app/javascript/components/products/cardProduct.scss [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    color: $black;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    &__title {
+      @extend %text-properties;
+      margin: 6px 0 2px;
+
+      @media only screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: check if we already have a variable for this resolution

File: app/javascript/components/products/cardProduct.scss [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+      }
+    }
+
+    &__subtitle {
+      color:$grey;
+      font-size: 12px;
+      font-weight: 900;
+      line-height: 1.2;
+
+      @media only screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: check if we already have a variable for this resolution

File: app/javascript/components/products/cardProduct.scss [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+        font-size: 14px;
+      }
+    }
+
+    &__price {
+      @extend %text-properties;
+      color: $grey;
+      margin: 2px 0 11px;
+
+      @media only screen and (min-width: 700px) {
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: check if we already have a variable for this resolution

File: app/javascript/components/products/cardProduct.scss [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+      border-radius: 4px;
+      border: none;
+      font-family: $font-stack;
+      font-size: 9px;
+      font-weight: 900;
+      height: 17px;
+      line-height: 1.22px;
+      outline: none;
+      width: 116px;
+      transition: all .5s ease;
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: sort these properties

File: app/javascript/components/products/cardProduct.scss [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+      &:hover {
+        transition: all .5s ease;
+        font-size: 16px;
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: sort these properties

File: app/javascript/components/products/cardProduct.scss [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+      }
+
+      @media only screen and (min-width: 700px) {
+        font-size: 14px;
+        height: 35px;
+        width: 170px;
+        cursor: pointer;
```
Comment (OswaldoPineda) [inline app/javascript/components/products/cardProduct.scss]: sort these properties

File: app/javascript/components/products/products.scss [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+    flex-wrap: wrap;
+    justify-content: space-around;
+    margin: 0 auto;
+    width: 100%;
+  }
+}
+
+.products-header {
+  display: flex;
+  align-items: baseline;
```
Comment (OswaldoPineda) [inline app/javascript/components/products/products.scss]: sort these properties

**Review by OswaldoPineda — CHANGES_REQUESTED** [review]
nice code, I only have some comments related to best practices, good luck!

File: app/javascript/components/products/Products.js (hunk 1 above)
Comment (JuanequeX) [reply app/javascript/components/products/Products.js]: Done my bro

File: app/javascript/components/products/Products.js (hunk 2 above)
Comment (JuanequeX) [reply app/javascript/components/products/Products.js]: Done

File: app/javascript/components/products/cardProduct.scss (hunk 3 above)
Comment (JuanequeX) [reply app/javascript/components/products/cardProduct.scss]: Done

**Review by OswaldoPineda — APPROVED** [review]
LGTM

**Pull request merged**